tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
This comment also doesn't add any value. This is a policy which leads to comments like the ones above. I'll add it now, but I want this be discussed now, since you have introduced this "policy" without any discussion in the team. /cc @chirimoya @wachterjohannes 

As already said I have a VERY strong opinion against that.